bgpd/bgp_route.c

```diff
@@ -2066,10 +2066,13 @@ int subgroup_process_announce_selected(struct update_subgroup *subgrp,
 	struct attr attr;
 	afi_t afi;
 	safi_t safi;
+	struct bgp *bgp = NULL;
```

Comment:
No need to set a value here since you set it on line 2075; if that line is ever removed or incorrectly changed then this will suppress a compiler warning that would otherwise catch an uninitialized value